config/common.yaml

```diff
@@ -228,7 +228,126 @@ console:
         tls:
           insecureEdgeTerminationPolicy: Redirect
           termination: passthrough
-
+## KIE smartrouter BEGIN
+smartrouter:
+  persistentVolumeClaims:
+    - metadata:
+        name: "{{.ApplicationName}}-smartrouter-claim"
+        labels:
+          app: "{{.ApplicationName}}"
+          service: "{{.ApplicationName}}-smartrouter"
+      spec:
+        accessModes:
+          - ReadWriteMany
+        resources:
+          requests:
+            storage: "64Mi"
+  deploymentConfigs:
+    - metadata:
+        name: "{{.ApplicationName}}-smartrouter"
+        labels:
+          app: "{{.ApplicationName}}"
+          service: "{{.ApplicationName}}-smartrouter"
+      spec:
+        strategy:
+          type: Recreate
+        triggers:
+          - type: ImageChange
+            imageChangeParams:
+              automatic: true
+              containerNames:
+                - "{{.ApplicationName}}-smartrouter"
+              from:
+                kind: ImageStreamTag
+                namespace: "openshift"
+                name: "rhpam{{.Version}}-smartrouter-openshift:{{.ImageTag}}"
+          - type: ConfigChange
+        replicas: 2
```

Comment:
Is this right / default?

Reply:
Yes, it is the common value for prod and immutable-monitor